src/app/authentication/security-http-interceptor.ts

```diff
@@ -16,14 +16,14 @@ export class SecurityHttpInterceptor implements HttpInterceptor {
 
     return next.handle(request)
       .catch(errorResponse => {
-        if (errorResponse.status !== 401) {
+        if (errorResponse.status !== 403) {
```

Comment:
If we return on `401` now, we won't log the user out (which happens at the bottom of this function). We should probably handle 401, 403 and potentially even 500 in here, and allow execution to return if the response is anything else.

Reply:
I think we still need the code here to handle the 401 by calling logout()